packages/blocks/chart/src/base.css

```diff
@@ -0,0 +1,248 @@
+/* –––––––––– Root variables –––––––––– */
```

Comment:
[optional] It'd be nice to scope the CSS if possible. Otherwise, we'll end up with style leaks like here:
https://blockprotocol.org/@kickstartds/blocks/button

Or maybe we can ignore this and wait till we have a proper sandboxing

Reply:
Yeah, I just copied in the style-sheet provided by Maggie - maybe given this Chart block isn't styled yet I should remove it entirely? WDYT?

Reply:
If we don't use the styles and the file is saved somewhere else, we can delete it I guess yes. Otherwise we should keep in mind that the block is not compatible with the hub yet.